<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>osmDirectoryConversionClass</class>
 <widget class="QWidget" name="osmDirectoryConversionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>osmDirectoryConversion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择目录：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inDirInputEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="inFileDialogBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transformBtn">
         <property name="text">
          <string>转换</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>输出目录：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outDirInputEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="outFileDialogBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateBtn">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeWidget" name="infileTree">
       <column>
        <property name="text">
         <string>文件名</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="outFileList"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="osmDirectoryConversion.qrc"/>
 </resources>
 <connections/>
</ui>
